packages/components/src/PaymentAccordion/Component.purs

```diff
@@ -0,0 +1,170 @@
+module KSF.PaymentAccordion.Component where
```

Comment:
Oh and we can just move this file right under `/components/src` and call it `KSF.PaymentAccordion`. The `ComponentName/Component.purs` is a convention we're moving out from.

Reply:
Done.